docs/core/reference/changelog.md

```diff
@@ -10,6 +10,26 @@ For updates to subpackages, see below:
 - [Node.js SDK](https://github.com/chain/chain/blob/1.1-stable/sdk/node/CHANGELOG.md)
 - [Ruby SDK](https://github.com/chain/chain/blob/1.1-stable/sdk/ruby/CHANGELOG.md)
 
+<a name="1.1.5"></a>
```

Comment:
It was a nice exercise to write them here. I've got a branch pasting these to the right places in `main`; I don't mind them in the historical branch as well right now.